pkg/notes/notes_test.go

```diff
@@ -105,3 +105,38 @@ func TestNoteTextFromString(t *testing.T) {
 	result, _ = NoteTextFromString("```release-note\ntest\n```")
 	require.Equal(t, "test", result)
 }
+
+func TestGetPRNumberFromCommitMessage(t *testing.T) {
+	testCases := []struct {
+		name             string
+		commitMessage    string
+		expectedPRNumber int
+	}{
+		{
+			name: "Get PR number from merged PR",
+			commitMessage: `Merge pull request #76030 from andrewsykim/e2e-legacyscheme
+
+    test/e2e: replace legacy scheme with client-go scheme`,
+			expectedPRNumber: 7630,
```

Comment:
```suggestion
			expectedPRNumber: 76030,
```

Reply:
Good finding, seems the `t.Parallel()` is problematic here as it hid that mistake